sim/core/base_stats.go

```diff
@@ -433,13 +433,13 @@ var ClassBaseStats = map[proto.Class]map[int]stats.Stats{
 	proto.Class_ClassDruid: {
 		25: {
 			stats.Health:      138,
-			stats.Mana:        199,
+			stats.Mana:        479,
```

Comment:
Think you might still need to pull in the change to keep this value low and instead alter how the base cost value is used? Otherwise total Mana values will be incorrect due to the first 20 points issue.

Reply:
I had thought that initially but it turns out there's code here that already accounts for that so I just adjusted the base number, reverted the base cost change, and I think it's working correctly:

https://github.com/wowsims/wotlk/blob/sod-fork/sim/core/mana.go#L42